eng/scripts/Setup-MQTTContainer.ps1

```diff
@@ -6,13 +6,43 @@ param (
 )
 
 if ($IsLinux -and $AgentImage -match "ubuntu") {
-    docker pull azsdkengsys.azurecr.io/eclipse-mosquitto:2.0.1
+    $FullCurrentPath = Get-Location | Resolve-Path
+    
+    # Setup certificates
+    $CAPass = 1122
+
+    # CA Generation
+    Invoke-Expression "sudo openssl genrsa -passout pass:$($CAPass) -des3 -out ca.key 2048"
+    Invoke-Expression "sudo openssl req -new -x509 -days 1826 -key ca.key -out ca.pem -subj `"/O=CA-cert/CN=127.0.0.1`" -passin pass:$($CAPass)"
```

Comment:
A recent change to argument parsing for PowerShell might necessitate a lot of `Invoke-Expression` but if possible just put the expressions in directly and see if things execute properly. (Test locally to maintain sanity in a tight iterative loop.) 

```suggestion
    sudo openssl req -new -x509 -days 1826 -key ca.key -out ca.pem -subj "/O=CA-cert/CN=127.0.0.1" -passin pass:$($CAPass)
```

It's possible that in the expressions with quotes you may need to supply more quotes (something like single quotes surrounding `'"/O=CA-cert/CN=127.0.0.1"'`) 


Most expressions below without quotes should just work.